# Build swapimport.exe since it is undesirable to use Google's and it is not present without running hooks
# Requires https://github.com/Eloston/syzygy

--- a/chrome/chrome_exe.gypi
+++ b/chrome/chrome_exe.gypi
@@ -8,7 +8,7 @@
       # GN version: //chrome
       'target_name': 'chrome',
       'type': 'none',
-      'dependencies': [ 'chrome_initial', ],
+      'dependencies': [ 'chrome_initial', '<(DEPTH)/third_party/syzygy/syzygy_swapimport.gyp:syzygy_swapimport', ],
       'conditions': [
         ['OS == "win"', {
           'actions': [
